FlowersOrder/FlowersOrderScreen/FlowersOrderTableViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TableViewCell.swift
+//  FlowersOrder
+//
+//  Created by Dragos Cosma on 03/02/2020.
+//
+
+import UIKit
+
+class FlowersOrderTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var smallImage: UIImageView!
+    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
```

Comment:
The cell's UI (on the storyboard) looks good, but maybe we don't need those nested stackviews. Instead, we can use constraints to just position the labels and keep everything flat.